parlai/core/params.py

```diff
@@ -92,6 +92,18 @@ def fix_underscores(args):
     return args
 
 
+class CustomHelpFormatter(argparse.ArgumentDefaultsHelpFormatter):
+    """
+    https://stackoverflow.com/questions/18275023/dont-show-long-options-twice-in-print-help-from-argparse
```

Comment:
Produces a custom formatted --help option.

See https:// for more information.